crates/cadenza-markdown/src/generated/snapshots/cadenza_markdown__generated__test_data__code_block_params__code_block_params_ast.snap

```diff
@@ -0,0 +1,12 @@
+---
+source: crates/cadenza-markdown/src/generated/test_data.rs
+expression: "# Interactive Example\n\nThis example shows code with parameters.\n\n```cadenza editable hidden\nlet setup = initialize()\n```\n\nThe code above is hidden and editable.\n\n```javascript\nconsole.log(\"Hello\");\n```\n\nDifferent languages are supported!\n"
+---
+[
+    [#, "Interactive Example"],
```

Comment:
I think I would prefer to just add the synthetic token types we need rather than use these tokens. Otherwise it's going to be really difficult to define macros in cadenza code to handle these callbacks. We want parsing markdown to be pretty much first-class anyway so I think it's fine. The tokens are just u16 so we've got plenty of space for it.